src/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { movies } from './MoviesGrid';
+
+let ErrorBoundary = (props: any) => {
```

Comment:
To define, that the function is React functional component, you can write the following:
```ts
let ErrorBoundary: FC = (props) => {...}
```
In the case above `props.children` will be recognized by Typescript.
If you want to additionally define some custom props, you can write the following:
```ts
interface ErrorBoundaryProps {
  propA: string,
  propB: Boolean,
  ...
}
let ErrorBoundary: FC = ({ propA, propB }) => {
   // And you can access props
   console.log(propA)
}
```

Reply:
Refactored.